src/property-filter/__tests__/property-filter-token-editor.test.tsx

```diff
@@ -361,7 +361,9 @@ describe.each([false, true] as const)('token editor labels, isMobile = %s', isMo
       expect(editor.propertyField().findControl()!.find('button')!.getElement()).toHaveAccessibleName(
         'Property string'
       );
-      expect(editor.operatorField().findControl()!.find('button')!.getElement()).toHaveAccessibleName('Operator =');
+      expect(editor.operatorField().findControl()!.find('button')!.getElement()).toHaveAccessibleName(
+        enableTokenGroups ? 'Operator =' : 'Operator = Equals'
+      );
```

Comment:
Previous version was not correct.

This is how `enableTokenGroups=true` look like

<img width="588" alt="image" src="https://github.com/user-attachments/assets/78f72fae-2b77-4c9b-8d30-5fd6a3526300" />

This is `enableTokenGroups=false`

<img width="255" alt="image" src="https://github.com/user-attachments/assets/b82a9521-96e2-4f20-b786-2c97efff3a21" />

The "Equals" string is indeed there and the test should assert that.

The issue has been exacerbated by our `i18nStrings` mock not providing the `operatorEqualsText` like it should